packages/terra-modal/lib/Modal.scss

```diff
@@ -0,0 +1,34 @@
+@import '~terra-mixins';
+
+.terra-Modal {
+  background-color: #fff;
+  border-radius: 5px;
+  bottom: 10px;
+  display: block;
+  left: 10px;
+  max-height: 100%;
+  outline: none;
+  overflow: hidden;
+  position: fixed;
+  right: 10px;
+  top: 10px;
```

Comment:
We should be using rem units. When the rem mixin is available we can start using pixel units in the mixin and output rems.